kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/index.vue

```diff
@@ -381,8 +381,7 @@
       // TODO semantic string names
       save: 'Save',
       // TODO: Handle singular/plural
-      selectionInformation:
-        '{count, number, integer} of {total, number, integer} resources selected',
+      selectionInformation: '{count, number, integer} resources selected',
```

Comment:
Unfortunately we can't edit strings like this due to string freeze. Ping me if you need help